homeassistant/components/plum_lightpad/light.py

```diff
@@ -64,6 +94,16 @@ def name(self):
         """Return the name of the switch if any."""
         return self._load.name
 
+    @property
+    def device_info(self):
+        """Return the device info."""
+        return {
+            "name": self.name,
+            "identifiers": {(DOMAIN, self.unique_id)},
```

Comment:
I think the `llid` or `Logical Load ID` might be a better choice here - they are big long GUIDs.